src/App.jsx

```diff
@@ -14,70 +16,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isSelected, setIsSelected] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
The name is a bit confusing as sounds like a boolean value
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/dc2e4258-0236-488c-b3c8-fad6b71f393b)